### PR TITLE
Hack to skip haskell submissions

### PR DESCRIPTION
**Pull request by lnussel**
It's just too much and needs to be submitted manually

### CODE REVIEW

File: update_crawler.py [hunk 1]
```diff
@@ -307,6 +310,13 @@ def crawl(self, packages):
         sources = self.get_source_infos(self.from_prj, packages)
         self.update_targets(targets, sources)
 
+    def get_devel_project(self, package):
```
Comment (jberry-suse) [inline update_crawler.py]: drop if `StagingAPI` used

File: update_crawler.py [hunk 2]
```diff
@@ -270,8 +270,11 @@ def update_targets(self, targets, sources):
             else:
                 targetinfo = targets[package]
 
-                #if package != 'openssl':
-                #    continue
+                # XXX: make more generic :-)
+                devel_prj, _ = self.get_devel_project(package)
```
Comment (jberry-suse) [inline update_crawler.py]: perhaps `StagingAPI.get_devel_project(FACTORY, package)`

**Comment by lnussel** [comment]
I don't understand the travis failure, works locally

**Comment by jberry-suse** [comment]
Failure appears to the same one in all test results #807.

**Pull request merged**